src/modules/roc_netio/target_uv/roc_netio/iclose_handler.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_netio/target_uv/roc_netio/iclose_handler.h
+//! @brief close handler.
```

Comment:
Nitpick: we usually start a `@brief` with a capital letter.

Reply:
Fixed